samples/contrib/azure-samples/databricks-pipelines/pipeline_cli.py

```diff
@@ -0,0 +1,85 @@
+import os
+import sys
+import logging
+import kfp
+import fire
+
+logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.INFO)
+
+class MyCLI:
+    """
+    CLI for Kubeflow Pipelines.
+
+    This CLI allows us to compile and run our pipelines in Kubeflow Pipelines without accessing
+    Kubeflow portal.
+    """
+
+    @staticmethod
+    def compile(
+            path
+    ):
+        """
+        Compile a Kubeflow pipeline.
+
+        Args:
+            path: Path to the pipeline (e.g. databricks_run_pipeline.py)
+        """
+
+        logging.info("Compiling '%s'...", path)
+        compiled_path = f"{path}.tar.gz"
+        result = os.system(f"dsl-compile --py {path} --output {compiled_path}")
+        if result != 0:
+            logging.error("Failed to compile '%s' with error code %i.", path, result)
+            sys.exit(result)
+
+        return compiled_path
+
+    @staticmethod
+    def run(
+            path,
+            host,
+            params={}
+    ):
+        """
+        Run a compiled pipeline in Kubeflow.
+
+        Args:
+            path: Path to the compiled pipeline (e.g. databricks_run_pipeline.py.tar.gz)
+            host: Kubeflow server address (e.g. http://localhost:8080)
+            params: Pipeline parameters (e.g. '{\"run_name\":\"test-run\",\"parameter\":\"10\"}'
+        """
+
+        logging.info("Running '%s' in '%s'...", path, host)
+        client = kfp.Client(f"{host}/pipeline")
```

Comment:
Note that whether the `/pipeline` suffix is needed depends on the type of deployment of KFP. If it's an inverse proxy host this will cause error. Probably it's better to let user provide the full host address?

Reply:
I've removed /pipeline from the code and now user can provide it if needed.

Reply:
Thanks!